### ISSUE TITLE
no-unnecessary-waiting does not get flagged when using a const as an argument to cy.wait()

### ISSUE THREAD

**Issue by stealthrabbi** [cypress-io/eslint-plugin-cypress#296]
Flagged by plugin as rule violation of `cypress/no-unnecessary-waiting`:

```ts
cy.wait(500);
```

Permitted. Does not get flagged: 
```ts
const waitTimeMs = 500;
cy.wait(waitTimeMs);
```

This was found with 5.2.0 of the plugin.

**Comment by stephenwade** [comment]
@stealthrabbi I had the same issue and I fixed it by updating eslint-plugin-cypress. Make sure you're using the latest version of the plugin.
